### PR TITLE
[cinder] Redo non-vcenter-volume agents

### PR DESCRIPTION
**Pull request by fwiesel**
Most options are simply passed through,
so the logic within the charts is fairly limited.

We need to work around NFS shares though,
and mount them with kubernetes.

### CODE REVIEW

**Comment by fwiesel** [comment]
Mounting is completely delegated to kubernetes for now, until we have an image with sapcc/loci#16

**Comment by hemna** [comment]
I am not sure the etc/_volume.conf.tpl file is even used.  I can't see the output from that template in the deployed cinder pods.   I think the vcenter operator generates the cinder-volume.conf file from the 
templates/vcenter_datacenter_cinder_configmap.yaml

**Comment by fwiesel** [comment]
Exactly, the  vsphere volumes from where you describe. 
The changes are related to the "static" configuration via the values files, which we haven't used for ages.

In the past, it was triggered by the values in
```
global:
  volumes_netapp: []
  volumes: []
```
The `global.volumes` was my first attempt some time ago to generalise the `global.volumes_netapp`.
Since they are not used, I propose to remove them and replace with something new.

To trigger those templates, you need to define a volume in a value file. The corresponding PR is in our internal repo for qa-de-1 (and the deployed code as well).

File: openstack/cinder/values.yaml [hunk 1]
```diff
@@ -72,6 +69,22 @@ cinderApiPortPublic: 443
 cinderApiPortInternal: 8776
 cinderApiPortAdmin: 8776
 
+defaults:
+  backends:
+    common: {}
+    cinder.volume.drivers.netapp.common.NetAppDriver:
```
Comment (joker-at-work) [inline openstack/cinder/values.yaml:75]: Should we consolidate the key `backend_defaults` from above into this structure (as a follow-up maybe)?

**Review by joker-at-work — APPROVED** [review]
In general, it looks like we're only supporting static NFS volumes here. Not sure, but maybe that should be reflected in the name.

Overall, no blockers from my side.

File: openstack/cinder/values.yaml (hunk 1 above)
Comment (fwiesel) [reply openstack/cinder/values.yaml:75]: Yes, I think, that would make sense. I can create a PR for that.

Comment (fwiesel) [reply openstack/cinder/values.yaml:75]: I had another look, and I am not sure, how to integrate it.
The defaults in this section are the defaults for the settings for an individual backend.
The `backend_defaults` are the `backend_defaults` section, which apply to all backends of a host, but only vmware.
It definitely needs some rework, but I am unclear a bit on how to do that.

Comment (joker-at-work) [reply openstack/cinder/values.yaml:75]: I thought we could maybe do something like `defaults.backends."cinder.volume.drivers.vmware.vmdk.VMwareVcVmdkDriver".backend_defaults` or something - but not sure either. If we don't have a good way, then we leave it until we figure it out $somewhen.

**Pull request merged**